azure-pipelines.yml

```diff
@@ -421,18 +415,30 @@ jobs:
     condition: eq(variables.bitstreamStrategy, 'cached')
     displayName: Prepare cached bitstream
   - bash: |
-      set -e
+      set -ex
+      . util/build_consts.sh
       module load "xilinx/vivado/$(VIVADO_VERSION)"
-      ci/scripts/build-bitstream-vivado.sh top_earlgrey cw310
+      SUB_PATH=hw/top_earlgrey
+      mkdir -p $OBJ_DIR/$SUB_PATH $BIN_DIR/$SUB_PATH
+      ci/bazelisk.sh build //hw/bitstream/vivado:standard --define bitstream=vivado
+      cp -rLvt $OBJ_DIR/$SUB_PATH/ \
+        $($REPO_TOP/bazelisk.sh outquery-all //hw/bitstream/vivado:fpga_cw310)
+      cp -rLvt $BIN_DIR/$SUB_PATH/ \
+        $($REPO_TOP/bazelisk.sh outquery-all //hw/bitstream/vivado:standard)
+
+      # Rename bitstreams to be compatible with subsequent steps
+      # TODO(#13807): replace this after choosing a naming scheme
+      mv $BIN_DIR/$SUB_PATH/lowrisc_systems_chip_earlgrey_cw310_0.1.bit $BIN_DIR/$SUB_PATH/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.orig
+      mv $BIN_DIR/$SUB_PATH/fpga_cw310_rom.bit $BIN_DIR/$SUB_PATH/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.splice
```

Comment:
Nit: The shellcheck living inside my brain says we should quote all these file paths :)